Source/Common/Config.cpp

```diff
@@ -489,14 +500,18 @@ fextl::string GetDataDirectory() {
   return DataDir;
 }
 
-fextl::string GetConfigDirectory(bool Global) {
+fextl::string GetConfigDirectory(bool Global, const PortableInformation& PortableInfo) {
+  const char* ConfigOverride = getenv("FEX_APP_CONFIG_LOCATION");
+  if (PortableInfo.IsPortable && (Global || (!Global && !ConfigOverride))) {
```

Comment:
`!Global` is implied by... `Global` being false:

```suggestion
  if (PortableInfo.IsPortable && (Global || !ConfigOverride)) {
```